src/app/filter-column/filter-column.model.ts

```diff
@@ -464,6 +469,40 @@ export class Filter {
     return filters;
   }
 
+  private static createFilterFromJsonFilter(filter: any, jsonFilter: any) {
+    filter.type = jsonFilter.type;
+    filter.participantColumn = jsonFilter.participantColumn;
+    filter.parentName = jsonFilter.participantColumn.object;
+    let selectedOptions = [];
+    if (filter.selectedOptions != null && jsonFilter.options != undefined) {
+      for (let value of Object.values( jsonFilter.options )) {
+        if (value['value'] && filter.selectedOptions.includes(value['value'])) {
+          selectedOptions.push(value['value']);
+        }
+      }
+    }
+    if (filter.filter1 == undefined || filter.filter1 == null) {
+      filter.filter1 = new NameValue(filter.participantColumn.name, null);
+    }
+    filter.filter1.value = this.replace(filter.filter1.value);
+    let newFilter = new Filter(filter.participantColumn, filter.type, jsonFilter.options, filter.filter2, filter.range, filter.exactMatch, filter.filter1,
+      selectedOptions, (filter.filter1 == null || filter.filter1 == undefined) ? null : filter.filter1.value,
```

Comment:
you have already checked in above if statement that if `filter.filter1` is null or undefined you set value on it, I think that at this point `filter.filter1`  will always have value.